legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/sqlQueryToString/SQLQueryToString.pure

```diff
@@ -773,19 +785,31 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::proce
    $children->map(child | processJoinTreeNode($child, $tableAlias->cast(@TableAlias), $dbConfig, $format, $extensions))->joinStrings($val, '', '');
 }
 
+
+function <<access.private>> meta::relational::functions::sqlQueryToString::isDefaultSchema(table:Table[1], dbConfig : DbConfig[1]):Boolean[1]
+{
+  ($table.schema.name->length() > 0) && ($table.schema.name == 'default');

```

Comment:
Do we need this part of the check?
($table.schema.name->length() > 0)

Reply:
Good catch. Will remove.